pkg/ephemeral-disk/ephemeral-disk_test.go

```diff
@@ -53,37 +49,6 @@ var _ = Describe("ContainerDisk", func() {
 		return f.Close()
 	}
 
-	AppendEphemeralPVC := func(vmi *v1.VirtualMachineInstance, diskName string, claimName string, backingDiskIsblock bool) {
-		vmi.Spec.Domain.Devices.Disks = append(vmi.Spec.Domain.Devices.Disks, v1.Disk{
-			Name: diskName,
-			DiskDevice: v1.DiskDevice{
-				Disk: &v1.DiskTarget{},
-			},
-		})
-		vmi.Spec.Volumes = append(vmi.Spec.Volumes, v1.Volume{
-			Name: diskName,
-			VolumeSource: v1.VolumeSource{
-				Ephemeral: &v1.EphemeralVolumeSource{
-					PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
-						ClaimName: claimName,
-					},
-				},
-			},
-		})
-
-		By("Creating a backing image for the PVC")
-		Expect(createBackingImageForPVC(diskName, backingDiskIsblock)).To(Succeed())
-
-		// Test the test infra itself: make sure that the backing file has been created.
```

Comment:
While it's not very pretty to begin with, I noticed we dropped this bit in this PR.
I think it may be worth to keep, maybe as part of createBackingImageForPVC

Reply:
Other than that, the changes are equivalent, so looks good (maybe other than the memory bit but that's totally irrelevant for the unit that's being tested)

Reply:
+1 to Alexs comments

Reply:
@akalenyu 
I dropped it because there is no need for it anymore
```go
	AppendEphemeralPVC := func(vmi *v1.VirtualMachineInstance, diskName string, claimName string, backingDiskIsblock bool) {
		vmi.Spec.Domain.Devices.Disks = append(vmi.Spec.Domain.Devices.Disks, v1.Disk{
			Name: diskName,
			DiskDevice: v1.DiskDevice{
				Disk: &v1.DiskTarget{},
			},
		})
		vmi.Spec.Volumes = append(vmi.Spec.Volumes, v1.Volume{
			Name: diskName,
			VolumeSource: v1.VolumeSource{
				Ephemeral: &v1.EphemeralVolumeSource{
					PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
						ClaimName: claimName,
					},
				},
			},
		})
```
all this could be replaced with `libvmi` package
```go
vmi := libvmi.New(libvmi.WithEphemeralPersistentVolumeClaim("fake-disk", "fake-pvc"))
```
which is the main task
Did I get it right ??

Reply:
You did, I meant the bit about `Test the test infra`

Reply:
> You did, I meant the bit about `Test the test infra`

So do I need to make any change in this PR or it is already approved ??

Reply:
Alex suggested the code you removed from line 77-84 to maybe move it into createBackingImageForPVC

Reply:
get it